client/utils/atomic.c

```diff
@@ -13,7 +14,7 @@ switch(sizeof(*object)) { \
 	case 4: \
 		return _InterlockedExchange((volatile long*)object, desired); \
 	case 8: \
-		return InterlockedExchange64((volatile LONG64*)object, desired); \
+		return _InterlockedExchange64((volatile long long*)object, desired); \
```

Comment:
shouldn't be volatile __int64 instead volatile long long?
microsoft documentation says:
```
__int64 _InterlockedExchange64_rel(  
   __int64 volatile * Target,  
   __int64 Value  
);
```

Reply:
P.S. i used InterlockedExchange64 instead of _InterlockedExchange64 because (i don't know why) visual studio warnings was telling me _InterlockedExchange64 is undefined... (this was happening with your stdatomic too)

Reply:
As I know, there is no difference between long long and __int64.
And yes, InterlockedExchange64 is not defined in the Visual Studio

Reply:
microsoft use char for _InterlockedExchange8, short for _InterlockedExchange16 and long for _InterlockedExchange but __int64 for _InterlockedExchange64 it’s strange, maybe there is some difference...

Reply:
i accept but recheck it please!